app/views/confirmation.scala.html

```diff
@@ -31,9 +31,7 @@
     </div>
 
     <p id="confirmation-email-message">@messages("confirmation.paragraph.confirmationEmail", confirmation.emailAddress)</p>
-    <ul class="list">
-        <li><a id="confirmation-pdf-link" target="_blank" href="@routes.ApplicationController.applicationPdf(confirmation.reference, Some(pdfToken))">@messages("confirmation.download.link.PDF")</a></li>
-    </ul>
+            <a id="confirmation-pdf-link" target="_blank" href="@routes.ApplicationController.applicationPdf(confirmation.reference, Some(pdfToken))">@messages("confirmation.download.link.PDF")</a>
```

Comment:
Can we change the indentation here to match the surrounding code?

Reply:
Thanks David, I have fixed the indentation issue.